tt_eager/tt_dnn/op_library/sharded/sharded_op.cpp

```diff
@@ -67,9 +67,9 @@ operation::ProgramWithCallbacks Sharded::create_program(
     auto& output_tensor = output_tensors.at(0);
 
     if (this->sharded_op_type == ShardedOpType::InterleavedToSharded) {
-        return interleaved_to_sharded_multi_core(input_tensor, output_tensor);
+        return interleaved_to_sharded_multi_core(input_tensor, output_tensor, 1, 0);
```

Comment:
Do you really need 1, 0 to be specified here? It looks like some magic numbers, bit confusing..

Reply:
Can put them in default args